Conch/ui/src/views/Login.js

```diff
@@ -11,7 +11,7 @@ export default {
                 m("legend", t("Login to Conch")),
                 m("input[type=text]", {
                     oninput: m.withAttr("value", Auth.setUsername),
```

Comment:
If the field is going to be the email address now, we probably should also change the variable names so that 'Auth.username' isn't secretly an email address.

Reply:
Captured in LP-43864914, I'll do the rename and make a new PR.